logdna-rails.gemspec

```diff
@@ -1,24 +1,22 @@
-# coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
+# frozen_string_literal: true
+
+lib = File.expand_path("lib", __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "version"
 
 Gem::Specification.new do |spec|
-  spec.name          = 'logdna-rails'
+  spec.name          = "logdna-rails"
   spec.version       = "#{LogDNA::VERSION}"
-  spec.authors       = 'Vilya Levitskiy'
-  spec.email         = 'vilya.levitskiy@logdna.com'
+  spec.authors       = "Gun Woo Choi, Derek Zhou, Vilya Levitskiy, Muaz Siddiqui"
+  spec.email         = "support@logdna.com"
 
-  spec.summary       = 'LogDNA Rails Logger'
-  spec.homepage      = 'https://github.com/logdna/rails'
-  spec.license       = 'MIT'
+  spec.summary       = "LogDNA Rails Logger"
+  spec.homepage      = "https://github.com/logdna/rails"
+  spec.license       = "MIT"
 
-  spec.files         = ["lib/logdna-rails.rb"]
-  spec.bindir        = 'exe'
-  spec.require_paths = ['lib']
+  spec.files         = `git ls-files -z`.split("\x0").select { |f| f.match(/(.md|.rb|.gemspec|Gemfile|LICENSE)$/) }
```

Comment:
Revert this to the way it was. There is no need to make it depended on git.

Reply:
@vilyapilya, that's how it's in the `Best Practices` - even in [`logdna/ruby`](github.com/logdna/ruby), it was with `git ls-files -z` - that's the recommended way of specifying files inside `gemspec` + some ownership-related files were missing initially

Reply:
I have a pr that changes it in Ruby. This is not a recommended way. That way it adds a redundant prerequisite of having git for creating the gem, but there is no need to have git there. Besides, there is no need to make so completed regex, it is a small file

Reply:
As it says [here](https://bundler.io/v1.16/guides/creating_gem.html#getting-started), it's using `git ls-files` when it builds and releases the gem; so, it's not creating any extra redundant prerequisite on the client-side and the place where it's getting built and released already has `git`

And [here](https://bundler.io/v1.16/guides/creating_gem.html#getting-started), it explains why it's not just `.rb` files to include

The reason of having `git` here is to pick up only committed files and prevent failures, as said.

Reply:
Gemspec is used when the gem is installed. And this creates a requirement for a user to have git initiated in the folder where they're trying to install git. The only file we need to include is the logdna-rails.rb. It is sufficient just to write this file explicitly.

Reply:
It is good to use git in the circleci scripts, git is the integral part of it. But git should not be in the prereq for building dependencies.